bats/tests/k8s/traefik.bats

```diff
@@ -90,10 +95,44 @@ assert_traefik_pods_are_up() {
     wait_for_apiserver
     # Check if the traefik pods come up
     try --max 30 --delay 10 assert_traefik_pods_are_up
-    run try --max 30 --delay 10 curl --head "http://$(get_host):80"
+}
+
+@test 'curl traefik from distro' {
+    run try --max 30 --delay 10 curl --head "http://localhost:80"
     assert_success
     assert_output --regexp 'HTTP/[0-9.]* 404'
-    run try --max 30 --delay 10 curl --head --insecure "https://$(get_host):443"
+    run try --max 30 --delay 10 curl --head --insecure "https://localhost:443"
     assert_success
     assert_output --regexp 'HTTP/[0-9.]* 404'
```

Comment:
Wouldn't all this turn into:

```
 curl_exe_traefik "http://localhost:80"
 curl_exe_traefik --insecure "https://localhost:443"
```

Reply:
That one is calling curl, basically, curl from inside the distro.